hyper-scale/race
# Add proof of concept for file upload api

Introduce example of route `/api/uploads` that returns url of uploaded file.
File uploads to firebase storage. In dev environment it upload to firebase storage emulator.

Manual testing in dev environment:

1. Call `/api/uploads` endpoint:

```
:~/Repo/hyper-scale/race$ curl -F 'file=@file.txt' localhost:3000/api/uploads
{"url":"https://firebasestorage.googleapis.com/v0/b/default-bucket/o/1639676180949_file.txt?alt=media&token=4e3bc6d5-3beb-4aaa-947b-387d8171209b"}
```

2. Verify file is uploaded:
    File is available in  http://localhost:4000/storage (firebase storage emulator ui).

This is just proof of concept and requires some more polishing and unit testing (separate PR).
